overlord/snapstate/autorefresh.go

```diff
@@ -157,8 +157,14 @@ func canRefreshOnMeteredConnection(st *state.State) (bool, error) {
 	if err != nil && err != state.ErrNoState {
 		return false, err
 	}
-
-	return onMetered != "hold", nil
+	switch onMetered {
+	case "hold":
+		return false, nil
+	case "force":
+		return true, nil
+	default:
+		return false, nil
```

Comment:
There's new feedback https://forum.snapcraft.io/t/snap-refresh-over-metered-connections/5001/30
It looks like we should do something like:
```
// holding refreshes on metered connections is an opt-in on Core devices,
// but a default on classic systems
return !release.IsOnClassic, nil
```